src/main/java/org/apache/accumulo/access/AccessEvaluatorImpl.java

```diff
@@ -86,6 +86,9 @@ static Collection<List<byte[]>> convert(Authorizations authorizations) {
   AccessEvaluatorImpl(final Collection<List<byte[]>> authorizationSets) {
 
     for (final List<byte[]> auths : authorizationSets) {
+      if (auths == null) {
```

Comment:
why allow nulls here?

Reply:
There was a test failing in apache/accumulo#4206. I think the issue was in the case when the user does not have any authorizations (and maybe no authorizations on the data).